src/CoreRT.WebAssembly.MonoBridge/CoreRT.WebAssembly.MonoBridge.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>8</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.JSInterop" Version="3.1.0" />
```

Comment:
Is this used? Or do we expect this to be used?

If not, we might be able to just throw the InternalCalls into CoreLib.

Reply:
Dont need  it for this at least, so have removed.